src/ghdldrv/ghdlvpi.adb

```diff
@@ -37,7 +37,7 @@ package body Ghdlvpi is
       --  Compute install path
       Ghdllocal.Set_Exec_Prefix_From_Program_Name;
 
-      return Ghdllocal.Exec_Prefix.all & Directory_Separator & "include";
+      return Ghdllocal.Exec_Prefix.all & Directory_Separator & "include/ghdl";
```

Comment:
`/` might be problematic on Windows. However, we currently test on MSYS2 only, so it should work.

Reply:
Tristan already fixed it on master: 2036d713a73a91a8014d0d63201d795807e0c924